Gruntfile.js

```diff
@@ -1,6 +1,19 @@
 var _ = require('underscore');
 var jasmineCfg = require('./grunt/tasks/jasmine');
 
+function getTargetDiff () {
```

Comment:
I would add a comment explaining what this method does and I think is better to use it instead saving a execution on a variable.